### PR TITLE
[KD] Change the copy on 'About' Page

### LINKED ISSUE #59: On About page, swap out temporary app name with 'Aisle Be There'
## Summary
We've decided on an official name for the app: Aisle Be There. Now we can swap out the temporary name previously used as placeholder on the About page for the new official name.

## Acceptance criteria
- [x] Switch out all occurrences of 'Box Makers' on About page for 'Aisle Be There'.

## Notes
Sub-issue of https://github.com/the-collab-lab/tcl-69-smart-shopping-list/issues/14

### PR DESCRIPTION
**Pull request by piecanoe**
## Description

Previously, we used "Box Makers" as the placeholder name in the copy on the About page. Since then, we've decided that "Aisle Be There" is the official name of the application. This code swaps out occurrences of the placeholder name for the official name. 

Note: I also changed the copy on the page to include more puns.  Very open to changing this back to the original copy if we want to keep a more serious tone. Also open to any other copy changes.

## Related Issue

Closes #59
Sub-issue of #12

## Acceptance Criteria

- [x] Switch out all occurrences of 'Box Makers' on About page for 'Aisle Be There'.

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.

### Before

https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/122697843/d720a005-b3e4-4fc9-96eb-c909b5bfece3

### After

https://github.com/the-collab-lab/tcl-69-smart-shopping-list/assets/122697843/e4c0efba-cca6-4c49-a99a-38002d1305de

## Testing Steps / QA Criteria

Navigate to 'About' page. Read through the copy on the page to see that there are no occurrences of "Box Makers".

### CODE REVIEW

**Review by hsiangj — APPROVED** [review]
No box makers found on the page per QA. I'm open to the puns 😂

**Review by rmorabia — APPROVED** [review]
Love it, lol.

File: src/views/About.jsx [hunk 1]
```diff
@@ -8,53 +8,68 @@ export function About() {
...
-						<h2>Make some lists</h2>
+						<h2>Lending a list-ening ear</h2>
 						<p>
-							Level up your organization game by creating specialized lists for
-							all your needs.
+							Aisle Be There to level up your organization game! Create
+							specialized lists for all your kneads!{' '}
+							<span role="img" aria-label="bread">
+								🍞{' '}
+							</span>
```
Comment (mxmason) [inline src/views/About.jsx]: 🔍 Love this!  You could make a tiny little helper component to deduplicate all the `role="img"` and `aria-label`-ing:

```jsx
/**
 * Render an emoji with an accessible label.
 * @param {{ children: React.ReactNode, label: string }} props 
 */
function Emoji({ children, label }) {
	return (
		<span role="img" aria-label={label}>
			{children}
		</span>
	);
}

<Emoji label="Bread">🍞</Emoji>
```

**Review by mxmason — APPROVED** [review]
Love it!

**Pull request merged**